ads/widespace.md

```diff
@@ -26,11 +26,32 @@ limitations under the License.
 </amp-ad>
 ```
 
+```html
+<amp-ad width=320 height=320
+    type="widespace"
+    data-sid="f666bfaf-69cf-4ed9-9262-08247bb274e4"
+    data-demo-Gender="F"
+    data-demo-Country="SE"
+    data-demo-Region="W1U 8EW"
+    data-demo-City="Stockholm"
+    data-demo-Yob="1981"
+    >
+</amp-ad>
+```
+
 
 ### Required parameter
 
 - `data-sid`
 
+## optional demography parameters
+
+- `data-demo-Gender`
```

Comment:
FYI: this doesn't need to be `data-demo-Gender`, you can put `data-demo-gender` and it will also be converted to `data['demoGender']`. Up to your preference.